modules/administration/pages/troubleshooting/tshoot-autoinstallation.adoc

```diff
@@ -8,7 +8,6 @@ For autoinstallation to work, these packages are required:
 * [package]``pyOpenSSL``
 * [package]``rhnlib``
 * [package]``libxml2-python``
-* [package]``spacewalk-koan``
```

Comment:
I think in 5.0 none of these packages are required. 
I think we only need venv-salt-minion and its dependencies. 
Maybe we should just point to the client tools channel to be available

Reply:
I think you proposal makes sense.  I tried to fix the focus of the TS section.  I hope my phrases are understandable.  Can you please check again?